Userland/Libraries/LibGfx/TIFFGenerator.py

```diff
@@ -9,7 +9,7 @@
 from enum import Enum
 from collections import namedtuple
 from pathlib import Path
-from typing import List, Type
+from typing import Any, List, Type
 
```

Comment:
Commit message: s/have/has/, s/for tag/for tags/

Reply:
Thanks for pointing it out, but oh god, that's embarrassing.